schemas/2016-04-30-preview/Microsoft.Compute.json

```diff
@@ -0,0 +1,4781 @@
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/InstanceViewStatus"
+              }
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The resource status information."
+        }
+      },
+      "description": "The instance view of a virtual machine extension."
+    },
+    "VirtualMachineExtensionProperties": {
+      "type": "object",
+      "properties": {
+        "forceUpdateTag": {
+          "type": "string",
+          "description": "How the extension handler should be forced to update even if the extension configuration has not changed."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "The name of the extension handler publisher."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the extension handler."
+        },
+        "typeHandlerVersion": {
+          "type": "string",
+          "description": "The type version of the extension handler."
+        },
+        "autoUpgradeMinorVersion": {
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Whether the extension handler should be automatically upgraded across minor versions."
+        },
+        "settings": {
+          "oneOf": [
+            {
+              "type": "object"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Json formatted public settings for the extension."
+        },
+        "protectedSettings": {
+          "oneOf": [
+            {
+              "type": "object"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Json formatted protected settings for the extension."
+        },
+        "instanceView": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/VirtualMachineExtensionInstanceView"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The virtual machine extension instance view."
+        }
+      },
+      "description": "Describes the properties of a Virtual Machine Extension."
+    },
+    "VirtualMachineProperties": {
+      "type": "object",
+      "properties": {
+        "hardwareProfile": {
```

Comment:
"hardwareProfile", "storageProfile" and "networkProfile" are required properties in the previous. Can we confirm on the change?